### ISSUE TITLE
Linter always show timeout dialog on error

### ISSUE THREAD

**Issue by vzamanillo** [AtomLinter/linter-scalac#91]
Due to an incorrect condition introduced in #82

```javascript
if (e.message !== 'Process execution timed out') {
  atom.notifications.addInfo('linter-scalac: `scalac` timed out', {
    description: 'A timeout occured while executing `scalac`, it could be due to lower resources '
                 + 'or a temporary overload.',
  });
} else {
  atom.notifications.addError('linter-scalac: Unexpected error', { description: e.message });
}
```
